apps/src/applab/applab.js

```diff
@@ -1212,18 +1212,54 @@ Applab.createImageUploadButton = function (opts) {
                  divApplab.appendChild(newLabel));
 };
 
+var TURTLE_X_OFFSET = -15;
```

Comment:
Maybe add a quick comment that these are to center the turtle image over the turtle location?